crates/nu-cli/src/commands/into_int.rs

```diff
@@ -0,0 +1,79 @@
+use crate::commands::WholeStreamCommand;
+use crate::prelude::*;
+use nu_errors::ShellError;
+use nu_protocol::{Primitive, ReturnSuccess, Signature, SyntaxShape, UntaggedValue, Value};
+
+use num_bigint::ToBigInt;
+
+pub struct IntoInt;
+
+#[derive(Deserialize)]
+pub struct IntoIntArgs {
+    pub rest: Vec<Value>,
+}
+
+#[async_trait]
+impl WholeStreamCommand for IntoInt {
+    fn name(&self) -> &str {
+        "into-int"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("into-int").rest(SyntaxShape::Number, "the values to into-int")
+    }
+
+    fn usage(&self) -> &str {
+        "Convert value to integer"
+    }
+
+    async fn run(
+        &self,
+        args: CommandArgs,
+        registry: &CommandRegistry,
+    ) -> Result<OutputStream, ShellError> {
+        into_int(args, registry).await
+    }
+
+    fn examples(&self) -> Vec<Example> {
+        vec![Example {
+            description: "Convert filesize to integer",
+            example: "ls | into-int $it.size | = $it / 1024",
+            result: None,
+        }]
```

Comment:
Great! I think we're almost ready. The last thing you need is a test so that we can keep it working.

The easiest way to add a test to commands these days is to add an example and set the `result:` field to the value that's expected out.  For you, you could do a test that echoes a filesize. Here's one possible example

```
into-int 10kb | = $it / 1024
```

And then setting `result` to the answer for that.  Once you have that, we should be ready to land.

Thanks again for putting it together.

Reply:
How do I pass the filesize as an argument to the `into-int` command?

```
/home/user/utam0k/nushell(byte-int)> into-int 10kb | = $it / 1024
error: Type Error
  ┌─ shell:1:10
  │
1 │ into-int 10kb | = $it / 1024
  │          ^^^^ Expected number, found 10kb
```

Reply:
You may need to use SyntaxShape::Any for now

Reply:
I added it!

Reply:
umm.. why is the added test failing?

Reply:
I tried to fix it.